docs/manipulation.rst

```diff
@@ -315,6 +315,32 @@ and True elsewhere. It is this way to be consistent with ``astropy.nddata``.
 .. note:: The mask attribute is the only attribute modified by ``snr_threshold()``. To
              retrieve the masked flux data use ``spectrum.masked.flux_masked``.
 
+Shifting
+--------
+
+To shift a spectrum in wavelength, one can use a construct like this:
+
+.. code-block:: python
+
+    >>> from specutils import Spectrum1D
+    >>> wavelengths = np.arange(0, 10)*u.um
+    >>> flux = 100*np.abs(np.random.randn(10))*u.Jy
+    >>> spectrum = Spectrum1D(spectral_axis=wavelengths, flux=flux)
+    >>> >>> spectrum  #doctest:+SKIP
+    <Spectrum1D(flux=<Quantity [ 31.65751688, 115.6368968 , 118.58240303, 186.46500763,
+                163.31955247,  90.92780989,  22.69088175,  51.81322025,
+                 11.70522032,  63.81487901] Jy>, spectral_axis=<Quantity [0., 1., 2., 3.,
+                 4., 5., 6., 7., 8., 9.] um>)>
+    >>> wunit = spectrum.spectral_axis.unit
+    >>> shift = 1.23
+    >>> new_spec = Spectrum1D(spectral_axis=(spectrum.spectral_axis.value + shift) * wunit, flux=spectrum.flux, wcs=None)
```

Comment:
I think this example should demonstrate using a shift as a `Quantity`. There's a case to be made that a shift could be applied to a spectral axis that is not in the same unit, adding the quantity version of the spectral axis with a quantity version of a shift should implicitly transform them to get the sum

```suggestion
    >>> shift = 12300 * u.AA
    >>> new_spec = Spectrum1D(spectral_axis=spectrum.spectral_axis + shift, flux=spectrum.flux, wcs=None)
```

Reply:
Done